docs/getting-started/edfi-exchange/technology/ed-fi-ods-cloud-deploy-aws-suite-3/deployment-guide/launch-non-prod.md

```diff
@@ -248,9 +248,10 @@ If you have encountered any issue or would like some more information on the sol
 
 ## Next Steps
 
-Once your environment is up and running on AWS, information on using the software can be found in following the links in Step 8 above.
+Once your environment is up and running on AWS, information on using the software can be found in following the links in Step 8 above. For more information, also see:
 
-Further support on using the Ed-FI ODS/API solution can be obtained at the [Ed-Fi Alliance website](@site/docs/reference/ods-api-platform.mdx). In addition, technical support is available. Information can be found in the [AWS ODS/API Solution - Frequently Asked Questions page](./faq.md).
+* [Ed-Fi ODS/API Platform](/reference/ods-api-platform)
+* [AWS ODS/API Solution - Frequently Asked Questions page](./faq.md).
```

Comment:
The `@site` link loads the raw markdown, so we need to avoid that 😆 .

![image](https://github.com/user-attachments/assets/75c78e3f-b662-4900-882b-d7c3b22de73d)

Reply:
Ah good catch! Can't believe I missed that XD